charts/oes/values.yaml

```diff
@@ -447,6 +447,14 @@ stormdriver:
   image:
     repository: quay.io/opsmxpublic/stormdriver
     tag: v1.2.6
+####################################################
```

Comment:
Revert this changes.. few more files need to be updated

Reply:
Removed the create controller secret block of code. So please take care of it.